release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -764,14 +901,19 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
+|CSI Google Filestore Driver Operator
+|-
+|-
+|TP
+
 |CSI automatic migration
-(AWS EBS, Azure file, GCP disk, VMware vSphere)
+(Azure file, VMware vSphere)
 |TP
 |TP
 |TP
 
 |CSI automatic migration
-(Azure Disk, OpenStack Cinder)
+(AWS EBS, Azure Disk, GCP disk, OpenStack Cinder)
 |TP
 |GA
 |GA
```

Comment:
@lisa 
AWS EBS, OpenStack Cinder start GA from 4.11.
Azure Disk, GCP disk start GA from 4.12 not 4.11.

Reply:
# `@lisa` User Notification

Hello! You're receiving this notice because you've tagged me, @lisa, on GitHub, likely in error. Lisa is a common first name, and happens to be mine. I use the @lisa username here on GitHub. While being tagged in various repositories is an interesting way for me to discover new projects, it's likely not your intention. Did you mean to tag someone else? Did you mean to use `@lisa` instead, in a [word as word](https://en.wikipedia.org/wiki/Use%E2%80%93mention_distinction) usage?

# Next Steps

I'm unsubscribing from this issue, pull request or project so if you meant to tag someone else to notify them, I would recommend that. If you meant to use my user ID as a plain string, without having GitHub notify me, you can do so by wrapping with backticks (the \` character), like so: `@lisa`.

---

(Unless you actually did want my input, in which case, message me on Slack because I'm not familiar with this space)

Reply:
@lpettyjo ^^

Reply:
Good catch. Fixed.